File: src/main/java/me/andre111/d20server/message/util/ResponseOk.java
package me.andre111.d20server.message.util;

import me.andre111.d20server.message.Message;

public class ResponseOk extends Message {
	public final String to;
	
	public ResponseOk(Message message) {
		this.to = message.getClass().getSimpleName();
	}
}

File: core/common/message/util/response-ok.js
import { Message } from '../message.js';
import { registerType } from '../../util/datautil.js';

export class ResponseOk extends Message {
    to;

    constructor(to) {
        super();
        this.to = to;
    }

    getTo() {
        return this.to;
    }
}
registerType(ResponseOk);
